### PR TITLE
Case Studies Redesign

### PR DESCRIPTION
**Pull request by Prid13**
Here's the first fully working draft of the redesign Case Studies/Showcase pages.

- 3 Pages have been set up (Vimla, CKH IOD, MKI) with the template
- The Template "language"/syntax should (hopefully) be straightforward to use/edit (not finalized yet)
- Not mobile friendly yet, and tweaks still need to be made
- Can only access via URL right now (no button/dropdown menu)

Once it's been deployed to the dev-site, access the showcases via these links:

- **[Vimla](https://jstrieb.github.io/link-lock/#eyJ2IjoiMC4wLjEiLCJlIjoiM1lXY3dtVHFXLzMxZ3RZdk5aTFp1dmhyV2dGSFpqL1gwcUhwWGp3Wlk1cjJGdkhzOFFHMUs2eGdaSFN2T3VLemtoa05uWFB3c1ZGckMyQmt0TlYwIiwiaSI6IjgxeWwySmljVytoVXJBWm8ifQ==)**
- **[CKH IOD](https://jstrieb.github.io/link-lock/#eyJ2IjoiMC4wLjEiLCJlIjoiNlVtNWVhSEdjcU5zb3k0NmtqYWxIMEoreGNLeHJxaEJSZWgycVVvUHlLT2FnYlJMSXNzWlNrT2o5aDJ5VDVNTjh1K1lBcS9oL1VqQWpjVlJiV1JOMUJnPSIsImkiOiJ1WWR5Q0VkSFQ4ZDVJQ1d2In0=)**
- **[MKI](https://jstrieb.github.io/link-lock/#eyJ2IjoiMC4wLjEiLCJlIjoiSWJQdmo4a0E5TFZBMEd5MUNhSXVXMC9YaXRtWUlndzR6L2w4Z2FtUzJMdGs2OGw3bnJJa0NkVnhrOGZGV1ppV3ptN1FDcDVMODFjeTJkbkpsQT09IiwiaSI6ImV4UURUVjJJWEQxYkhrak4ifQ==)**

Example screenshot of the full template:

https://www.yogile.com/uh2lwb8ityj

### CODE REVIEW

File: website/src/pages/casestudies/index.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+---
+slug: casestudies
```
Comment (jonny-wg2) [inline website/src/pages/casestudies/index.md:2]: This needs updating. perhaps  borrowing the template from https://www.wgtwo.com/case-studies/

File: website/src/pages/casestudies/ckh-iod.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+import BigTitle from "./components/bigTitle";
+import CenteredTitle from "./components/centeredTitle";
+import Cards from "./components/cardContainer";
+import Card from "./components/card";
+
+<TwoColumnSection>
+
+  <Column sticky>
+    <BigTitle pillText="MVNE" titleText="CKH IOD">
+      There should be something here -- feels rather empty
```
Comment (jonny-wg2) [inline website/src/pages/casestudies/ckh-iod.md:29]: Let's add support for the company logo after the company name

**Comment by jonny-wg2** [comment]
Amazing work @Prid13 . Love how you were able to accomplish this with just .md. 

A couple feedback:
- [ ] rename all casestudy to `showcase`
- [ ] I created a draft PR for the new header, but if you could add the code from this into yours that would be great https://github.com/working-group-two/wgtwo.com/pull/164
- [ ] for wgtwo.com/showcase I see you are the index.md as a Vimla. Perhaps we could have a similar page to https://www.wgtwo.com/case-studies/ with all of the "showcases" there? Maybe for the sake of this PR though we could leave this blank and do another (don't want to feature creep)
- [ ] On the showcase page of a company it would be nice to have support for an image on the left side (the logo of the company just under the company name)
- [ ] On desktop view, could we try to get the company left side 1/3 and the challenge and solution boxes 2/3 of width (unsure what the look would be)
- [ ] On mobile probably best to make it all full width

**Comment by torotimes** [comment]
Hey @Prid13

Can you review and take action on @jonny-wg2's action list above? 

When that's done, we can start populating the showcase with final content. 

We'll probably wait a little bit with actually publishing the showcase, though, just for the record. We'd like to land a couple more customers before we make it live. But's let's complete the template and get things in shape! We're really close!

**Comment by Prid13** [comment]
On it 👍

**Comment by Prid13** [comment]
@jonny-wg2 you mean like this? :)

image

Check the logo. And tried 2/5 and 3/5 instead of 1/3 and 2/3, which would look like this:

image

Or is this better, you think? :)

**Comment by Prid13** [comment]
**DONE!** https://wgtwo-dev.netlify.app/showcase/

- Syntax for adding logo is `<BigTitle logoUrl="/img/logo_name.png">`
- The description for companies under `wgtwo.com/showcase` is pulled from the Markdown pages' frontmatter, specifically the `description` part. Any change should be made there
- @torotimes Sample File created for you to use as a starting point. Just copy it, rename to whatever you want, and end it with ".md". All the instructions are written inside :)
  - when making a page for a new company, an additional step needs to be taken, but we'll take care of that (adding the new company to `/pages/showcase/showcase-list.tsx`) ;)

Discussion:
- Should the descriptions under /showcase (the overview where all companies are shown) be translateable? They were prior to the change, but I'm wondering if they should still be?
- The dropdown menu in the navbar is static, and I hate it 😅 there's probably a hack to make it dynamic so that we avoid repeating ourselves in coding, but should we bother? If only a few companies are being added in total, and new ones will only be added every now and then, then perhaps it's not a big deal.

**Comment by jonny-wg2** [comment]
I updated the PR with some CSS style improvements and refined the text for the final deliverable.

Regarding your discussion points @Prid13, we will eventually want to get this translated but can do this in a second PR after merging. Regarding the auto-generate dropdown, unsure if there is an easy way to do this as it appears we need to hard code this in the docusarous config? Open for ideas though but again not needed for right now.

All in all, I believe this PR is good to go live. @torotimes can take it from here and give the final 👍🏻 ✅

**Comment by Prid13** [comment]
@torotimes Done :)

One problem. I wasn't sure which part was the Title/Heading of the Challenge part, so I opted for the first sentence only. But it looks out of place and doesn't match the "Challenge" heading:

Screenshot_6856

But adding the whole sentence makes it unwieldy large:

Screenshot_6854

**Review by torotimes — APPROVED** [review]

**Comment by torotimes** [comment]
On the "Challenge": i think it was the right choice, the way you did it, @Prid13  :) 

Will the final URL be https://www.wgtwo.com/showcase/ckh-iod/ ?

**Comment by torotimes** [comment]
Hi @Prid13!

Some edits from the team on the CKH IOD case study:

**Bullet point that says "Programmability":**
Change to "Automation and programmability"

**Remove this sentence: "Aggressively adding new countries and customers"**

**Update this box with the correct link:**
image
Visit the official page of CKH IOD 
New link: https://ckhiod.com

**Pull request merged**